query/sources/impl/src/main/java/org/codice/ddf/admin/sources/csw/persist/CreateCswConfiguration.java

```diff
@@ -67,17 +63,6 @@ public CreateCswConfiguration(ConfiguratorSuite configuratorSuite) {
 
     @Override
     public BooleanField performFunction() {
-        Configurator configurator = configuratorSuite.getConfiguratorFactory()
```

Comment:
Should we comment this out and leave a TODO stating we can't do this until the csw feature can start up properly? If so, we should do the same for all the sources. We can add an ignore tag to the tests as well.